docker/install_dependencies.sh

```diff
@@ -79,40 +61,20 @@ CARLA_TAR="CARLA_0.9.10.tar.gz"
 cd /home/carma/src/
 if [[ ! -f "$CARLA_TAR" ]]; then
     echo "!!! $CARLA_TAR not present in the installation directory, downloading automatically instead. This could take a long time, consider downloading the file manually and placing it in the installation directory. !!!"
-    wget "https://carla-releases.s3.eu-west-3.amazonaws.com/Linux/CARLA_0.9.10.tar.gz"
+    wget -q "https://carla-releases.s3.eu-west-3.amazonaws.com/Linux/CARLA_0.9.10.tar.gz"
 fi
 
 sudo mkdir -p /opt/carla
 sudo chown -R carma:carma /opt/carla
 tar xzvf "$CARLA_TAR" -C /opt/carla
 
 # Installation of Co-Simulation Tool
-wget "https://archive.apache.org/dist/maven/maven-3/3.8.3/binaries/apache-maven-3.8.3-bin.tar.gz"
-sudo mkdir -p /opt/maven
-sudo chown -R carma:carma /opt/maven
-tar xzvf apache-maven-3.8.3-bin.tar.gz -C /opt/maven
-export PATH=/opt/maven/apache-maven-3.8.3/bin/:$PATH
-
-# Build co-simulation tool
-cd /home/carma/src/co-simulation
-mvn clean install -DskipTests
-cd bundle/target
-sudo mkdir -p /opt/carma-simulation
-sudo chown -R carma:carma /opt/carma-simulation
-unzip carla-sumo-mosaic-22.1-SNAPSHOT.zip -d /opt/carma-simulation
-sudo chmod +x /opt/carma-simulation/mosaic.sh
-sudo mkdir /opt/carma-simulation/scenarios/tmp_scenario
-cp bundle-22.1-SNAPSHOT.jar /opt/carma-simulation
-
-# Deploy scenario files
-cd /home/carma/src/co-simulation
-unzip sample_scenario.zip -d /opt/carma-simulation/scenarios
-
-# Install NS-3
-cd "/opt/carma-simulation/bin/fed/ns3/"
-chmod +x ns3_installer.sh
-set -x
-./ns3_installer.sh -q
-sudo cp /home/carma/src/co-simulation/patch/run.sh /opt/carma-simulation/bin/fed/ns3
-
-echo "Build complete!!!"
+wget -q "https://archive.apache.org/dist/maven/maven-3/3.8.3/binaries/apache-maven-3.8.3-bin.tar.gz"
+sudo mkdir -p /usr/share/maven
+sudo chown -R carma:carma /usr/share/maven
+tar xzvf apache-maven-3.8.3-bin.tar.gz -C /usr/share/maven
+# Adding configuration file to fix error output from CARLA (https://github.com/carla-simulator/carla/issues/2820)
+set +H
+echo $'pcm.!default {\n  type plug\n  slave.pcm \"null\"\n}' | sudo tee /etc/asound.conf
```

Comment:
If this is CARLA specific, it should probably get grouped with the CARLA installation steps.

Reply:
I think it is a dependency of carla that spits out these log messages. Something about a sound driver or something. I can move it to the CARLA installation section